### PR TITLE
Added links to RDS loading data information

### PR DESCRIPTION
**Pull request by lynnlangit**
3 links - 1 to quick start for loading, 1 for Redshift table analyze, 1 for Redshift table vacuum

### CODE REVIEW

**Comment by jlevy** [comment]
Assigning to @max-zanko our Redshift editor for additional review, and we'll also await merge resolution.

File: README.md [hunk 1]
```diff
@@ -1142,6 +1142,7 @@ Redshift
 -	📒 [Homepage](https://aws.amazon.com/redshift/) ∙ [Developer guide](http://docs.aws.amazon.com/redshift/latest/dg/) ∙ [FAQ](https://aws.amazon.com/redshift/faqs/) ∙ [Pricing](https://aws.amazon.com/redshift/pricing/)
 -	**Redshift** is AWS’ [data warehouse](https://en.wikipedia.org/wiki/Data_warehouse) solution, which is highly parallel, share-nothing, and columnar. It is very widely used. It [was built](https://en.wikipedia.org/wiki/Amazon_Redshift) with [ParAccel](https://en.wikipedia.org/wiki/ParAccel) and [Postgres](https://en.wikipedia.org/wiki/PostgreSQL).
 
+
```
Comment (ThanosBaskous) [inline README.md]: Nit: Extra newline.

File: README.md [hunk 2]
```diff
@@ -1153,6 +1154,7 @@ Redshift
 -	It has a leader node and computation nodes (the leader node distributes queries to the computation ones). Note that some functions [can be executed only on the lead node.](https://docs.aws.amazon.com/redshift/latest/dg/c_SQL_functions_leader_node_only.html)
 -	🔸Redshift does not support many Postgres functions, most notable date/time related or aggregates. See the [full list here](https://docs.aws.amazon.com/redshift/latest/dg/c_unsupported-postgresql-functions.html).
 -	Major 3rd-party BI tools support Redshift integration (see [Quora](https://www.quora.com/Which-BI-visualisation-solution-goes-best-with-Redshift)).
+-       AWS has a useful tutorial to get you started [loading data](http://docs.aws.amazon.com/redshift/latest/dg/tutorial-loading-data.html) into Redshift.  It is important to understand the use of the table commands to [analyze](http://docs.aws.amazon.com/redshift/latest/dg/t_Analyzing_tables.html) and [vacuum](http://docs.aws.amazon.com/redshift/latest/dg/r_VACUUM_command.html) commands when loading data.
```
Comment (ThanosBaskous) [inline README.md]: Nit: two spaces between sentences - style in the rest of the guide is one space.

**Comment by ThanosBaskous** [comment]
Fantastic! Thank you @lynnlangit!

**Comment by ThanosBaskous** [comment]
LGTM pending section editor @max-zanko's approval (his requested changes not accepted yet.)

**Comment by lynnlangit** [comment]
I think I made all requested changes.  What is yet to be done here?

**Comment by ThanosBaskous** [comment]
Excellent, thanks @lynnlangit and @max-zanko! @lynnlangit, would you mind resolving the conflicts? Once that is done, I''ll go ahead and merge.

**Comment by lynnlangit** [comment]
This information appears to have already been added.

**Pull request closed without merging**